src/modules/ROOT/pages/changelog/202209/09.json

```diff
@@ -111,7 +109,7 @@
     },
     {
       "what": [
-        "user getting the error `“Could not update record: failed to complete transaction: Error 1366: Incorrect decimal value: `updated` for column `operation` at row 1”`, when adding or updating the dedicated record revision table in modules."
+        "indirectly updating the record revisions table when records were updated."
```

Comment:
Why remove the mention of the error? The change when this happened is good; let's keep the mention of the error also.